gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Sat Dec 12 17:01:19 CET 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
```

Comment:
Could you maybe use the `gradle-6.8.3-all.zip` here? So we get the Gradle documentation as well (see https://github.com/andOTP/andOTP/pull/719#issuecomment-792345363)?

Reply:
Done! I also updated to Gradle 6.9, which was released earlier today.